pkgs/applications/video/kodi/default.nix

```diff
@@ -153,7 +158,7 @@ in stdenv.mkDerivation rec {
       which
       pkgconfig gnumake
       autoconf automake libtool # still needed for some components. Check if that is the case with 18.0
-    ];
+    ] ++ lib.optional useWayland [ wayland-protocols ];
```

Comment:
Does this really belong to `nativeBuildInputs`?

Reply:
As I understand it, `wayland-protocols` are just xml files that describes the different protocols used in Wayland. They are converted to code at build time (usually with wayland-scanner for C/C++ headers).